### PR TITLE
Add Clout Price of Creator Coin in Profiles

### PR DESCRIPTION
**Pull request by carsenk**
Adds in the current $CLOUT price of a Creator's Coin on Profile Pages, also includes a new BitClout logo in svg format for use with theming or any other future needed icons.

Preview:
image

### CODE REVIEW

**Comment by carsenk** [comment]
@lazynina @maebeam @diamondhands0 needs review/merge 👍

**Comment by carsenk** [comment]
@lazynina ?

File: src/app/creator-profile-page/creator-profile-top-card/creator-profile-top-card.component.html [hunk 1]
```diff
@@ -106,9 +106,10 @@
 
       <div style="whitespace: nowrap">
         <div class="font-weight-bold" style="display: inline">
-          ~{{ globalVars.nanosToUSD(profile.CoinPriceBitCloutNanos, 2) }}
+          ~{{ globalVars.nanosToUSD(profile.CoinPriceBitCloutNanos, 2) }} (<strong>~<img src="assets/img/cloutlogo.svg" width="15px" class="clout" style="margin-right:-8px;margin-top:-2px;" />
```
Comment (lazynina) [inline src/app/creator-profile-page/creator-profile-top-card/creator-profile-top-card.component.html:109]: I would remove the `~` from within the parentheses.  It feels redundant to me.

**Review by lazynina — CHANGES_REQUESTED** [review]
Other than removing the `~` I like this change. will wait on @maebeam for a second opinion.

**Comment by carsenk** [comment]
Fixed and awaiting @maebeam

File: src/app/creator-profile-page/creator-profile-top-card/creator-profile-top-card.component.html [hunk 2]
```diff
@@ -106,9 +106,10 @@
 
       <div style="whitespace: nowrap">
         <div class="font-weight-bold" style="display: inline">
-          ~{{ globalVars.nanosToUSD(profile.CoinPriceBitCloutNanos, 2) }}
+          ~{{ globalVars.nanosToUSD(profile.CoinPriceBitCloutNanos, 2) }} (<strong><img src="assets/img/cloutlogo.svg" width="15px" class="clout" style="margin-right:-8px;margin-top:-2px;" />
+            {{ globalVars.nanosToBitClout(profile.CoinPriceBitCloutNanos, 2) }}</strong>)
         </div>
-        <div class="fc-muted" style="display: inline">Coin Price&nbsp;&nbsp;</div>
+        <div class="fc-muted" style="display: inline">Price&nbsp;&nbsp;</div>
```
Comment (maebeam) [inline src/app/creator-profile-page/creator-profile-top-card/creator-profile-top-card.component.html:112]: Let's keep the Coin Price copy for now

Comment (carsenk) [reply src/app/creator-profile-page/creator-profile-top-card/creator-profile-top-card.component.html:112]: Okay

**Pull request merged**